usaspending_api/broker/management/commands/load_broker_static_data.py

```diff
@@ -16,8 +16,9 @@ class Command(BaseCommand):
     @transaction.atomic
     def handle(self, *args, **options):
         logger.info("Starting data load: Broker Static Data")
-        logger.info("Loading External Data Types...")
 
+        # External Data Load Date
```

Comment:
What's the purpose of this inline code comment? What important information does it convey?

Reply:
Same as above. I don't mind removing in both cases.